src/integrationTest/resources/data/civil-daily-cause-list/civilDailyCauseList.json

```diff
@@ -23,7 +23,6 @@
             "courtRoomName": "1",
```

Comment:
Can we also do the same with the invalid list, and also the valid/invalid ones in the test folder too

Reply:
done.